src/fides/api/ops/util/connection_util.py

```diff
@@ -0,0 +1,243 @@
+from typing import List, Optional
```

Comment:
Code climate is pretty unhappy about this file. All of this code was cut/pasted from `connection_endpoints.py` to avoid circular reference errors. None of this code is actually new so it should be safe to merge.